src/app/modules/healthcheck/healthcheckRoutes.ts

```diff
@@ -1,8 +1,8 @@
-const HealthCheckController = require('./HealthCheckController')
+import {HealthCheckController} from './HealthCheckController'
```

Comment:
Any way to get the default export to be the class itself? `import HealthCheckController from './HealthCheckController'`

Reply:
We absolutely could. Having named exports just helps my auto-import plugin, but I think it still works if the default export matches the file name.

Should I change it?

Reply:
Eh I do like the idea of classes being 1:1 with files, and that's supported by the `export default class` pattern. The Typescript [docs](https://www.typescriptlang.org/docs/handbook/modules.html) say "If you’re only exporting a single class or function, use export default", so I'd lean in favor of that.